conformance/lang/expressions/trap_expr.balt

```diff
@@ -0,0 +1,252 @@
+Test-Case: output
```

Comment:
The structure we usually follow is to 
1. first cover the sections/sentences in the spec
2. then write different tests with different kinds of expressions

Please reorganize the tests to match that.

Reply:
Changed